.changelog/unreleased/bug-fixes/ibc-relayer/3954-interchainquery-missed-events.md

```diff
@@ -0,0 +1,3 @@
+- Correctly extract all `interchainquery` events if
+  there are more than one emitted at the same time
```

Comment:
```suggestion
- Fix a bug where Hermes would only ever extract the first emitted ICS 031 CrossChain Query event, which would cause it to miss the other CCQ events.
```